src/podman.scss

```diff
@@ -21,10 +21,18 @@
 }
 
 .container-block small {
-    flex-basis: calc(var(--pf-global--LineHeight--sm) * 1rem);
     line-height: var(--pf-global--LineHeight--sm);
     opacity: 0.7;
     overflow: hidden;
+    // line-clamp / max-lines needs webkit prefix, but is supported in modern all browsers like this
```

Comment:
Can't we use .pf-u-text-truncate class - instead of all this? 
We just want to truncate the text from what I understand.

Reply:
It needs to be clamped, as it can be arbitrary amounts of lines. This is the only way line-clamp works in browsers, sadly. Anything else is a hack.

Reply:
PF has a mixin for line clamping - see  pf-line-clamp in src/patternfly/sass-utilities/mixins.scss

Reply:
Seem you can use it like:
@include pf-line-clamp("1");

Reply:
Oh, that's interesting! Does it work for us though? (Or does it use inline CSS?)

Reply:
OK, I've looked into it and the CSS that's generated is similar, but it'll keep us from rewriting CSS hacks (which work across browser, but it still depends on nasty prefixed CSS even if it's supported well). 

I'll switch to using it in our CSS here instead.